cmd/consensus/main.go

```diff
@@ -316,7 +316,7 @@ func main() {
 				return fmt.Errorf("could not load beacon key file: %w", err)
 			}
 
-			rootEpoch := node.State.AtBlockID(node.FinalizedRootBlock.ID()).Epochs().Current()
+			rootEpoch := rootSnapshot.Epochs().Current()
 			epochCounter, err := rootEpoch.Counter()
```

Comment:
for consistency, could you rename this variable
```suggestion
			rootEpochCounter, err := rootEpoch.Counter()
```